File: Day 2/Python/Level 4/task5.py
def is_prime(number):
    if number < 0:
        return "Input positive numbers"
    elif number == 1:
        return "1 is not considered a prime number"
    elif number == 2 or number == 3:
        return "Prime number"
    else:
        for index in range (number):
            if number % index == 0:
                return "Not a prime number"
        return "Prime number"

File: Pre-Ileya/Day 2/Java/Level 4/Task5.java
public class Task5{
    public static void main(String[] args){
        System.out.println(isPrime());
    }

    public static String isPrime(int number){

        String status = "init";
        if number < 0 {
            status = "Input positive numbers";
            return status;
        }

        else if (number == 1) {
            status = "1 is not considered a prime number"; 
            return status;
        }

        else if (number == 2 || number == 3) {
            status = "Prime number";
            return status;
        } else {
            for (int index = 2; index < number; i++){
                if (number % index == 0){
                    status == "Not a prime number"
                    return status;
                }

            status == "Prime number";
            return status;
            }
        }
    }
}
